Userland/Utilities/lsirq.cpp

```diff
@@ -10,27 +10,51 @@
 #include <LibCore/System.h>
 #include <LibMain/Main.h>
 
+static ErrorOr<size_t> system_cpu_count()
+{
+    auto file = TRY(Core::File::open("/sys/kernel/cpuinfo", Core::OpenMode::ReadOnly));
+    auto buffer = file->read_all();
+    auto json = TRY(JsonValue::from_string({ buffer }));
+    return json.as_array().size();
+}
+
```

Comment:
Couldn't you get the same number from the number of entries in the per_cpu_call_counts array? It already is sliced to the amount of CPUs.

Reply:
Sure, let's do that! :)